catalog/UMT/UMT_Climate_IrrMapper_RF_v1_1.jsonnet

```diff
@@ -40,8 +40,11 @@ local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
     fields, 38,000 dryland fields, and over 500,000 square kilometers of
     uncultivated lands.
 
-    For version 1.1, the original training data was re-run on Landsat
-    Collection 2 and brought up-to-date.
+    For version 1.2, the original training data was greatly expanded, 
+    a RF model built for each state, and a more thorough validation and uncertainty 
+    analysis undertaken. See the [supplement](https://static-content.springer.com/esm/art%3A10.1038%2Fs43247-023-01152-2/MediaObjects/43247_2023_1152_MOESM3_ESM.docx) 
+    to our [paper](https://www.nature.com/articles/s43247-023-01152-2) on the impacts of irrigation on streamflow.
+
   |||,
   license: license.id,
```

Comment:
could you please add
'gee:skip_indexing': true,
at the top level? This is a temporary setting needed for the current process.